### DISCUSSION TITLE
Spacemouse USB, open path and silent callback

### DISCUSSION THREAD

**Discussion by hlprev** [JakubAndrysek/PySpaceMouse#15] [Ideas]
Hi! I did som changes to adapt your code to a project of mine (thanks btw). Would you be interested in including them in the master?

Checkout the fork:
https://github.com/hlprev/PySpaceMouse/tree/master/pyspacemouse

I added:

- another spacemouse module, spacemouse USB
- The possiblitiy to open a spacemouse by path
- a "silent" callback. For when to you callback, but not wanting it printing to terminal

**Comment by JakubAndrysek** [comment]
Hi, I have looked at [your repo](https://github.com/hlprev/PySpaceMouse/tree/master/pyspacemouse), but I didn't find any changes. Have you pushed them to GitHub?
The changes look promising, I will look at them.

Please, create the Pull request to this repo.

Thanks
Jacob

**Reply by hlprev** [reply]
My bad. I messed it up. But now I have created a pr with the changes. If you have some thoughts you wanna discuss I am all ears :).
